azDevOps/azure/templates/steps/build-pack-test-dotnet.yml

```diff
@@ -24,8 +24,9 @@ steps:
     inputs:
       command: 'restore'
       projects: '${{ parameters.package_path }}**/*.csproj'
+      # TEMP: Remove once published to nuget.org
       feedsToUse: 'select'
-      vstsFeed: '${{ parameters.package_feed }}'
+      vstsFeed: 'Stacks'
```

Comment:
why not just change the `parameters.package_feed` parameterto `stacks`?

Reply:
Good point. I can revert this change anyway, as the pipeline variable will overwrite this anyway.